backend/mailman_client/setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
```

Comment:
What does this file give us?

Reply:
This is the package manager. I separated it in case we wanted to make the client more robust and since the find_packages method goes through every .py file in the dir then I thought it might be worth separating in case we didnt wanna do that for other files.